src/Rand.php

```diff
@@ -1,28 +1,17 @@
 <?php
 /**
- * Zend Framework (http://framework.zend.com/)
- *
- * @link      http://github.com/zendframework/zf2 for the canonical source repository
- * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @link      http://github.com/zendframework/zend-math for the canonical source repository
+ * @copyright Copyright (c) 2005-2018 Zend Technologies USA Inc. (http://www.zend.com)
  * @license   http://framework.zend.com/license/new-bsd New BSD License
  */
 
 namespace Zend\Math;
 
-use RandomLib;
-
 /**
  * Pseudorandom number generator (PRNG)
  */
 abstract class Rand
 {
-    /**
-     * Alternative random byte generator using RandomLib
-     *
-     * @var RandomLib\Generator
-     */
-    protected static $generator = null;
```

Comment:
Since this is marked `protected`, it should stay in, but me marked `@deprecated`, with a note that it is no longer used internally.